sn_auditor/src/dag_db.rs

```diff
@@ -358,8 +360,24 @@ impl SpendDagDb {
                 .or_default()
                 .insert((addr, amount));
         } else {
+            // check with default key
+            if let Some(default_user_name_hash) =
+                spend.reason().get_sender_hash(&DEFAULT_PAYMENT_FORWARD_SK)
```

Comment:
We could reduce indentation and remove the return statement below with an `else if let`
```suggestion
        // check with default key
        } else if let Some(default_user_name_hash) =
                spend.reason().get_sender_hash(&DEFAULT_PAYMENT_FORWARD_SK)
```

And put the unknown participant block in a last else